lib/core/utils/get-scroll.js

```diff
@@ -19,17 +19,29 @@ axe.utils.getScroll = function getScroll(elm, buffer = 0) {
 	}
 
 	const style = window.getComputedStyle(elm);
-	const overflowXStyle = style.getPropertyValue('overflow-x');
+
+	// Checking Y before X since we assume that
+	// most scrolling is across the Y axis.
```

Comment:
These are just property lookups so I don't think splitting out the if statement with two different returns that both return the same thing is worth the millisecond savings.